termbox/src/lib.rs

```diff
@@ -216,6 +218,8 @@ impl Termbox {
     }
 
     pub fn present(&mut self) {
+        self.last_cursor = (0, 0);
```

Comment:
Hmm, so I guess because `(0, 0)` is not a valid location (upper-left corner is `(1, 1)`) this effectively invalidates the last location. Did I get this right?

Reply:
Yes. The position is (1, 1) based.